sql/moz-fx-data-shared-prod/accounts_backend_derived/users_services_daily_v1/query.sql

```diff
@@ -56,13 +56,10 @@ windowed AS (
     udf.mode_last(ARRAY_AGG(country) OVER w1) AS country,
     udf_contains_tier1_country(ARRAY_AGG(country) OVER w1) AS seen_in_tier1_country,
     LOGICAL_OR(event_name = 'reg_complete') OVER w1 AS registered,
-    ARRAY_AGG(event_name) OVER w1 AS service_events,
   FROM
     fxa_events
   WHERE
     DATE(submission_timestamp) = @submission_date
-    AND user_id_sha256 != ''
-    AND service != ''
```

Comment:
Removing this results in some new rows with empty `service` field. I'm not sure if we're not undercounting with this condition.

Reply:
We can remove it. I would like to understand how many additional rows this condition had removed, but I can do the comparison afterwards.